<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>item_new</class>
 <widget class="QDialog" name="item_new">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="weight_textbox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New Item</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="description_textbox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="price_textbox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="type_textbox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_textbox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
